lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -485,25 +487,36 @@ impl DomLayers {
         let front = DomScene::new(logger);
         let fullscreen_vis = DomScene::new(logger);
         let back = DomScene::new(logger);
+        let panel = DomScene::new(logger);
+
+        panel.dom.set_class_name("panel");
+        panel.dom.set_style_or_warn("pointer-events", "auto", logger);
+        panel.dom.set_style_or_warn("z-index", "0", logger);
+
+        back.dom.set_class_name("back");
+        back.dom.set_style_or_warn("pointer-events", "auto", logger);
+        back.dom.set_style_or_warn("z-index", "1", logger);
+
+        fullscreen_vis.dom.set_class_name("fullscreen_vis");
+        fullscreen_vis.dom.set_style_or_warn("z-index", "2", logger);
```

Comment:
I would like to understand the reasoning for z-indexes here. Why "panel" has z-index lower than other things here? So, for example, if we ever would like to show project chooser during working with nodes, should it appear below node visualizations?

Reply:
As mouse cursor is drawn somewhere above `back` layer (but below `front`), we have to use `z-index` of or `0`, `1`. I am afraid that using the same `z-index` for multiple layers would cause some issues with z-fighting, so I decided to make it bottom-most and increment `z-index`s of other layers by one. Making it bottom-most makes some sense to me, as it can be used for background element with fixed position.

Reply:
I forgot that the mouse cursor is in the `canvas` layer. Please add comment to the code of why this is on the back although logically it should be above nodes.

Reply:
Fixed documentation for struct field.